src/components/Hero/ContentHero/ContentHero.stories.tsx

```diff
@@ -25,9 +26,17 @@ const meta = {
 export default meta
 
 export const ContentHero: StoryObj = {
+  parameters: {
+    // Set asPath in mock router so the Breadcrums component can render
+    // the "home" text with correct translation
+    nextjs: {
+      router: {
+        asPath: "/en",
+      },
+    },
+  },
```

Comment:
A good to know here, as this is integrated into Storybook with NextJS.

https://storybook.js.org/blog/integrate-nextjs-and-storybook-automatically/#nextrouter

Reply:
Aha! interesting. Thanks